src/applications/vaos/tests/e2e/vaos-cypress-helpers.js

```diff
@@ -91,6 +91,7 @@ const mockUser = {
         '22-1990',
         '22-1990N',
         '22-1990E',
+        '22-1990S',
```

Comment:
I don't think this is necessary, we don't really read these values in our vaos tests. It's just copied and pasted from somewhere else.

Reply:
Ok I can go ahead and remove this.